testnet/src/tests/mempool.rs

```diff
@@ -0,0 +1,307 @@
+
```

Comment:
Please move these tests back into `src/chainstate/stacks/db/blocks.rs`.  They're not part of the testnet.

Reply:
I only moved the tests relying on the testnet frameworks. They are not part of the testnet, but they depend on this version of the testnet. If we're saying that the testnet is a separate module with its own dependencies, if we move this tests back to their initial location `core/mempool`(not `src/chainstate/stacks/db/blocks.rs`), we're introducing a dependency cycle (tesnet depends on core, core depends on testnet).